### ISSUE TITLE
about TTA

### ISSUE THREAD

**Issue by xrli-U** [EngrUsmaanAli/TopoTTA#2]
I am very interested in this paper and would like to discuss some questions with the authors. It seems that the code only trains the MLP on few-shot normal images. How is TTA reflected during testing? In the Implementation Details section of the paper, there is a sentence: “Optimisation uses Adam for 30 epochs with a learning rate of 10⁻³ on nominal samples and an effective batch size of one (weight decay 10⁻⁴ for stability).” Does this mean that the method only needs to be trained on a small number of normal samples from the training set?
